app/views/unattended/report_templates/subscription_-_general_report.erb

```diff
@@ -8,24 +8,36 @@ template_inputs:
   description: Search query to filter subscription entitlements. E.g. `quantity >
     0` to exclude subscriptions with unlimited entitlements. Keep unset for all subscriptions.
   advanced: false
+- name: Days from Now
+  required: true
+  input_type: user
+  description: "Show subscriptions expiring in <= this many days.  Select 'no limit' to show all entitlements."
```

Comment:
We no longer care about "entitlements"..

```suggestion
  description: "Show subscriptions expiring in <= this many days.  Select 'no limit' to show all subscriptions."
```

Reply:
Weird thing going on where I have updated the branch on my fork but the force push doesn't reflect on the PR.. 😕

Reply:
First time I have seen the "processing updates" spinner.. 😄 
![Screenshot from 2024-03-01 09-31-25](https://github.com/theforeman/foreman/assets/21146741/2b8ffacf-4426-48b7-b803-2d3b47def110)

Reply:
All good now..